Source/GoBot/GoBot.swift

```diff
@@ -1172,12 +1172,29 @@ class GoBot: Bot {
     }
 
     // MARK: Feeds
+    
+    /// The algorithm we use to filter and sort the home feed.
+    var homeFeedStrategy: FeedStrategy {
```

Comment:
I used to have a FeedStrategyBuilder with two functions buildHomeFeedStrategy and buildDiscoverFeedStrategy that I removed. But now that the strategies are set here, we might want ti bring it back and just have a var builder: FeedStrategyBuilder property here in GoBot. With that way, we continue to have a way to test the different strategies in unit tests (mocking the builder), and also remove the responsibility of knowing where the selected strategy is stored and place it in the builder.

Reply:
@martindsq that's not a bad idea. Maybe we can make that part of #564?

Reply:
Cool, I left a comment there to not forget about this